package.json

```diff
@@ -2,45 +2,49 @@
   "name": "ember-colpick",
   "version": "0.6.2",
   "description": "color picker component for ember using http://colpick.com/",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "Stefan Penner & Yapp Labs",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
+  "repository": "https://github.com/yapplabs/ember-colpick",
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
-    "test": "ember try:testall"
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
+    "test": "ember try:each"
   },
-  "repository": "https://github.com/yapplabs/ember-colpick",
-  "engines": {
-    "node": ">= 0.10.0"
+  "dependencies": {
+    "ember-cli-babel": "^6.6.0"
   },
-  "author": "Stefan Penner & Yapp Labs",
-  "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.2",
-    "ember-cli": "2.4.3",
-    "ember-cli-app-version": "^1.0.0",
-    "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-htmlbars": "^1.0.3",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
-    "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-qunit": "^1.4.0",
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-cli": "~2.18.2",
+    "ember-cli-dependency-checker": "^2.0.0",
+    "ember-cli-eslint": "^4.2.1",
+    "ember-cli-htmlbars": "^2.0.1",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
+    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-qunit": "^4.1.1",
     "ember-cli-release": "0.2.8",
```

Comment:
I wasn't sure whether to remove `ember-cli-release` or not? This isn't in the latest ember-cli package.json but unsure whether it's used in this addon specifically.

Reply:
Please do keep ember-cli-release. I like it for releasing new versions of addons.